apps/pythonapi/examples/numpy_example.py

```diff
@@ -0,0 +1,40 @@
+# %% [md]
+#
+# # Rendering Numpy data with Vapor
+#
+# Vapor supports rendering 2D and 3D numpy data.
+# In order to pass Numpy data to Vapor, create a data set of the type `vapor.dataset.PYTHON`
```

Comment:
The documentation says create a data set of type, `vapor.dataset.PYTHON`. But in the code below a convenience function, CreatePythonDataset(), is used to accomplish this task. The code and comment should be consistent.

Reply:
Fixed.